### PR TITLE
Add support for logs_archives resource

### PR DESCRIPTION
**Pull request by skarimo**
APITL-935

Note, this resource requires manual setup of AWS, GCP and Azure integration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,37 +206,38 @@ When running againts multiple destination organizations, a seperate working dire
...
-| users                                  | Sync Datadog users.                                      |
+| Resource                               | Description                                                       |
+|----------------------------------------|-------------------------------------------------------------------|
+| authn_mappings                         | Sync Datadog authn mappings.                                      |
+| dashboard_lists                        | Sync Datadog dashboard lists.                                     |
+| dashboards                             | Sync Datadog dashboards.                                          |
+| downtime_schedules                     | Sync Datadog downtimes.                                           |
+| downtimes (**deprecated**)             | Sync Datadog downtimes.                                           |
+| host_tags                              | Sync Datadog host tags.                                           |
+| logs_archives                          | Sync Datadog logs archives. Requires GCP, Azure, AWS integration. |
```
Comment (drichards-87) [inline README.md:217]:
```suggestion
| logs_archives                          | Sync Datadog logs archives. Requires GCP, Azure, and AWS integrations. |
```

Comment (drichards-87) [reply README.md:217]: Depending on intent, it may make more sense to use "or" instead of "and" here.

File: README.md [hunk 2]
```diff
@@ -256,6 +257,7 @@ See [Supported resources](#supported-resources) section below for potential reso
 | downtime_schedules                     | monitors                                                         |
 | downtimes (**deprecated**)             | monitors                                                         |
 | host_tags                              | -                                                                |
+| logs_archives                          | - (Requires manual setup of AWS, GCP and Azure integration)      |
```
Comment (drichards-87) [inline README.md:260]:
```suggestion
| logs_archives                          | - (Requires manual setup of AWS, GCP and Azure integrations.)      |
```

**Review by drichards-87 — DISMISSED** [review]
Left a couple of small suggestions from Docs and approved the PR.

**Review by amaskara-dd — APPROVED** [review]

**Pull request merged**